<?xml version="1.0"?>
<launch>
  <arg name="sim" default="false"/>

  <include file="$(find-pkg-share bitbots_bringup)/launch/base.launch" >
    <arg name="sim" value="$(var sim)"/>
  </include>

  <group unless="$(var sim)">
    <include file="$(find-pkg-share bitbots_ros_control)/launch/ros_control.launch" />
  </group>

  <arg if="$(env IS_ROBOT false)" name="taskset" default="taskset -c 3"/>
  <arg unless="$(env IS_ROBOT false)" name="taskset" default=""/>

  <node pkg="bitbots_dynamic_kick" exec="KickNode" output="screen" launch-prefix="$(var taskset)">
    <remap from="/kick_motor_goals" to="/DynamixelController/command"/>
    <rosparam command="load" file="$(find-pkg-share bitbots_dynamic_kick)/config/kick_config.yaml" />
    <rosparam command="load" file="$(find-pkg-share bitbots_dynamic_kick)/config/kick_config_unstable.yaml" />
  </node>
</launch>
